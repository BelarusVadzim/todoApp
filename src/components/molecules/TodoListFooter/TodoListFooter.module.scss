@import 'src/variables.scss';

.todoListFooter {
  align-items: center;
  color: $secondary-font-color;
  display: flex;
  font-size: $todoListFooter-font-size;
  justify-content: center;
  min-height: 4rem;
}

