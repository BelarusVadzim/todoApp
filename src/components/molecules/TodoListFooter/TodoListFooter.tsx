import React from 'react';
import { TextPlace } from 'components/atoms';
import style from './TodoListFooter.module.scss';

const TodoListFooter: React.FC = () => (
  <TextPlace className={style.todoListFooter}>
    Drag and drop to reorder list
  </TextPlace>
);
export default TodoListFooter;
