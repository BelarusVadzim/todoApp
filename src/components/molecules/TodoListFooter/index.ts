import TodoListFooter from './TodoListFooter';

export { TodoListFooter };