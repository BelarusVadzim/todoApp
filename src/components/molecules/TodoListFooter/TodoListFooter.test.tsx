import { render } from '@testing-library/react';
import TodoListFooter from './TodoListFooter';

jest.mock('components/atoms', () => ({
  Label: jest.fn(({ className, children }) => {
    return <div className={className}>{children}</div>;
  }),
}));

describe('<TodoListFooter />', () => {
  it('should render properly', () => {
    const component = render(<TodoListFooter />);

    expect(component.asFragment()).toMatchSnapshot();
  });
});
