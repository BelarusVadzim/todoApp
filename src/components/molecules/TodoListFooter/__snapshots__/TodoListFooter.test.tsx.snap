// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TodoListFooter /> should render properly 1`] = `
<DocumentFragment>
  <div
    class="todoListFooter"
  >
    Drag and drop to reorder list
  </div>
</DocumentFragment>
`;
